nsv13/code/modules/overmap/knpc.dm

```diff
@@ -311,16 +297,16 @@ This is to account for sec Ju-Jitsuing boarding commandos.
 	if(!..())
 		return 0
 	var/mob/living/carbon/human/H = HA.parent
-	var/obj/A = H.get_active_held_item()
+	var/obj/item/gun/G = H.get_active_held_item()
 	//We already have a gun
-	if(A && istype(A, /obj/item/gun))
+	if(G && istype(G))
 		return 0
```

Comment:
I mean

Reply:
There's literally nothing wrong with this. At worst it's cleaner and at best it's faster, depending on DM's compiler side optimizations

Reply:
always look at the last line of code
```suggestion
		return FALSE
```
it's just cosmetic
but it's everywhere

Reply:
I didn't even change that line, I'd rather keep things consistent with the rest of the file than just changing one random line. Feel free to make a PR addressing the whole file though, it's just out of scope for this PR